src/graphics.c

```diff
@@ -2342,9 +2445,12 @@ GdipGetNearestColor (GpGraphics *graphics, ARGB *argb)
 GpStatus WINGDIPAPI
 GdipSetPageScale (GpGraphics *graphics, REAL scale)
 {
-	if (!graphics) 
+	if (!graphics || scale <= 0.0 || scale > 1000000032) 
```

Comment:
I do not know where this number comes from... GDI+ accepts `1000000032` but rejects anything above it...